configuration/concepts/labOrders.csv

```diff
@@ -0,0 +1,37 @@
+uuid,Members,Answers,Fully specified name:pt_BR,Fully specified name:en,Short name:pt_BR,Short name:en,Data class,Data type,_version:1,_order:1002
+5a7222fa-2d30-4806-86a1-12b4c9d9eb26,,,Carga Viral,Carga Viral,Carga Viral,Carga Viral,LabTest,Numeric,,
```

Comment:
Eish I only spotted this afterwards, but @krishnanspace you need to add the correct English translations for these concepts as found in the excel spreadsheet on the ticket. Please revisit this. Thanks.